ros2launch/ros2launch/command/launch.py

```diff
@@ -145,6 +169,15 @@ def main(self, *, parser, args):
         else:
             raise RuntimeError('unexpected mode')
         launch_arguments.extend(args.launch_arguments)
+
+        if args.secure is not None:
```

Comment:
Is there a reason we're only comparing to `None` here?

```suggestion
        if args.secure:
```

Reply:
Yes, secure is set to `None` when the user does not provide `--secure`. If they provide it, but don't specify a keystore, it gets set to `''` instead